src/components/onboarding-done/onboarding-done.stories.js

```diff
@@ -0,0 +1,115 @@
+import { ChevronRight, ExternalLink, X } from 'lucide-react';
+import ProgressSteps from '../progress-steps';
+import Topbar from '../topbar';
+import OnboardingDone from './onboarding-done';
+import Button from '../button';
+import Label from '../label';
+
+export default {
+	title: 'Templates/OnboardingDone',
+	component: OnboardingDone,
+	parameters: {
+		layout: 'left',
+	},
+	tags: [ 'autodocs' ],
+	argTypes: {
+		children: {
+			description: 'Content to render inside the OnboardingDone.',
+			control: { type: 'none' },
+		},
+	},
+};
+
+const Template = ( args ) => {
+	return (
+		<OnboardingDone
+			{ ...args }
+			className="bg-gray-100 min-h-screen w-full pb-10"
```

Comment:
Use color from defined colors (Figma/Tailwind config).

❌ `bg-gray-100`

Reply:
It's of their choice, that's why not added in component level but from story level, giving a hint they can pass anything. It must be some custom one.

Reply:
Since we are giving the option to add className it's clear that the users/developers can use class names to apply different styles. IMO we should follow and match the mockup.

Reply:
Yes, matched without using pre defined classnames.

Reply:
Modified.